crates/analytics/src/clickhouse.rs

```diff
@@ -85,12 +85,14 @@ impl ClickhouseClient {
                 |t| Err(ClickhouseError::ResponseNotOK(t)).into_report(),
             )
         } else {
-            Ok(response
-                .json::<CkhOutput<serde_json::Value>>()
-                .await
-                .into_report()
-                .change_context(ClickhouseError::ResponseError)?
-                .data)
+            let ckh_output: CkhOutput<serde_json::Value> =
+                match serde_json::from_str(response.text().await.unwrap_or_default().as_str()) {
+                    Ok(output) => output,
+                    Err(_) => {
+                        return Err(ClickhouseError::ResponseError.into());
+                    }
+                };
+            Ok(ckh_output.data)
```

Comment:
This change was necessary as `reqwest` made `serde_json` to not be accessible as it is an internal feature which is used here at `analytics` crate (`.json`)

Reply:
https://docs.rs/reqwest/latest/reqwest/struct.Response.html#method.json

> Available on crate feature `json` only.

Moreover, we have been pulling the `json` feature in other crates which use `reqwest`.